libs/use-multi-media-streams/src/utils.ts

```diff
@@ -21,31 +24,40 @@ export const createStream = async ({
       return {
         id: mediaStream.id,
         stream: {
+          bitrate,
+          codec,
+          // TODO: better display screen label
+          label: label ?? mediaStream.getVideoTracks()[0].label ?? mediaStream.id,
           mediaStream,
+          simulcast: codec === 'h264',
           state: initialStreamState,
           type,
         },
       };
     }
 
     case StreamTypes.LOCAL: {
-      if (!localFile) {
+      if (!objectUrl) {
         return;
       }
 
       // Custom type as captureStream is not defined in HTMLVideoElement by default
       const video = document.createElement('video') as HTMLVideoElementWithCaptureStream;
```

Comment:
We should move this out in the future, and use the video-view to generate the stream.  Let's keep it for now.

Reply:
We will fix this in #274